sql/starrocks/desc.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<desc xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../_scripts/desc.xsd">
+
+    <!-- antlr-version 4.10 required because caseinsensitive is used in
+    the grammar. -->
+    <antlr-version>^4.10</antlr-version>
+
+    <!-- entry-point required because there are multiple start symbols
+    possible. -->
+    <entry-point>sqlStatements</entry-point>
+
+    <!-- Works with all targets because there is no target-specific
```

Comment:
The comment is now incorrect. It should read: "Works only with Java because of symbol conflicts."

Reply:
My mistake, I fix it and update the new desc.xml file.

Reply:
@kaby76 Thank you for your review. This is my first pr and i really appreciate your help. Thank you so much!